README.md

```diff
@@ -123,12 +155,97 @@ The nsx_certificate.crt file generated as output from the above command contains
+2. **password**: The password to authenticate with the NSX manager
+
+For example:
+```yaml
+- hosts: localhost
+  tasks:
+    - name: Update Tier0
+      nsxt_policy_tier0:
+        hostname: "default"
+        username: admin
+        password: my-password
+        validate_certs: False
+        display_name: test-tier0-1
+        state: present
+```
+
+##### Prinicipal Identity
+By specifying the following fields in the playbook:
+1. **nsx_cert_path**: Path to the certificate created for the Principal Identity using which the CRUD operations should be performed
+2. **nsx_key_path**: Path to the certificate key created for the Principal Identity using which the CRUD operations should be performed
+
+For example:
+```yaml
+- hosts: localhost
+  tasks:
+    - name: Update Tier0
+      nsxt_policy_tier0:
+        hostname: "default"
+        nsx_cert_path: /root/com.vmware.nsx.ncp/nsx.crt
+        nsx_key_path: /root/com.vmware.nsx.ncp/nsx.key
+        validate_certs: False
+        display_name: test-tier0-1
+        state: present
+```
+
+##### vIDM
+When NSX-T is configured to use VMware Identity Manager (vIDM) for authentication, you can supply an Authorization header with an authentication type of *Remote*. The header content should consist of a base64-encoded string containing the username@domain and password separated by a single colon (":") character, as specified in RFC 1945 section 11.1.
+
+For example, to authenticate a request using the credentials of user jsmith@example.com with password Sk2LkPM!, include the following key:value pair under **request_headers** in the playbook::
+- Authorization: 'Remote anNtaXRoQGV4YW1wbGUuY29tOlNrMkxrUE0h'
+
+For example:
+```yaml
+- hosts: localhost
+  tasks:
+    - name: Update Tier0
+      nsxt_policy_tier0:
+        hostname: "default"
+        request_headers:
+          Authorization: 'Remote anNtaXRoQGV4YW1wbGUuY29tOlNrMkxrUE0h'
+        validate_certs: False
+        display_name: test-tier0-1
+        state: present
+```
+
+##### SSL Verification
+You can use the flag *validate_certs* to perform SSL verification. You can also specify the path to a CA bundle using the paratemer *ca_path* in the playbook.
+
+For example:
+```yaml
+- hosts: localhost
+  tasks:
+    - name: Update Tier0
+      nsxt_policy_tier0:
+        hostname: "default"
+        nsx_cert_path: /root/com.vmware.nsx.ncp/nsx.crt
+        nsx_key_path: /root/com.vmware.nsx.ncp/nsx.key
+        validate_certs: True
+        ca_path: /path/to/my/ca-bundle
+        display_name: test-tier0-1
+        state: present
+```
+
+# Unit Testing
+To test the Ansible modules or see examples of playbooks, please put the respective playbook from unit_tests folder to the base folder and run the ansible-playbook.
+
+Please note that you must specify the correct vmware args in order to successfully update the resources.
+
 # Interoperability
 
 The following versions of NSX are supported:
 
  * NSX-T 2.4
- * Ansible 2.7
+ * Ansible 2.8.1
```

Comment:
Not sure why we had Ansible version here. It should list NSX version. NSX 2.4, NSX 2.5.x